### DISCUSSION TITLE
Sequence series for corrections

### DISCUSSION THREAD

**Discussion by christian-eriksson** [ledgersmb/LedgerSMB#7462] [Ideas]
Some countries require corrective invoices and/or transactions to have their own sequential series (and often unbroken series, at least per period).

[example](https://www.internationaltaxreview.com/article/2ba2yh4dbaiktmapruqdc/local-insights/spain-some-clarifications-on-the-issuing-of-corrective-invoices)

The suggestion here is to allow users to configure that corrective invoices and transactions have their own respective sequence series (i.e. one for invoices and one for transactions).

If users choose to activate the feature, corrective invoices and corrective transactions would booked under the configured sequential series. The series should not be editable on the transaction/invoice.

When users configure the sequence they should be able to set a post or prefix for the sequence (like for the custom sequences under `System -> Sequences` today).

**Comment by ehuelsmann** [comment]
I'm unsure what the problem statement in this idea is. That is to say, I have no idea in what way the existing functionality is I sufficient to handle the requirements from the quoted regulatory body.
